.github/workflows/release.yml

```diff
@@ -9,13 +9,15 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
+
```

Comment:
Is this accidental spacing? (If intentional that is fine).

Reply:
Wasn't accidental but I guess it doesn't really matter.